lib/build.rb

```diff
@@ -28,7 +28,9 @@ def initialize(data)
     end
 
     def run
-      @run_file = Tempfile.new("executor")
+      FileUtils.mkdir_p(config.jobs_dir) unless File.exists?(config.jobs_dir)
+
+      @run_file = Tempfile.new("executor", config.jobs_dir)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.